src/App.tsx

```diff
@@ -10,28 +10,28 @@ import GoogleLoginSuccess from './pages/google/successLogin';
 import About_us from './pages/about/about_us';
 import Seller from './components/seller/seller';
 import routes from './utils/routes';
+import Navigation from './components/navigation/nav';
+import { isLoggedIn } from './helpers/auth';
+import '../styles/index.css';
 
 function App() {
   library.add(fab, fas);
   return (
-    <div data-testid="app">
+    <div data-testid="app" className="mt-10">
       <Provider store={store}>
-        <Router>
-          <Routes>
-            <Route
-              path={`${import.meta.env.VITE_BACKEND_URL}/auth/google/success`}
-              element={<GoogleLoginSuccess />}
-            ></Route>
-            <Route path={routes.authenticate} element={<Accounts />}></Route>
-            <Route path={routes.signup} element={<Accounts />}></Route>
-            <Route path={routes.home} element={<Home />}></Route>
-            <Route path={routes.login} element={<Accounts />}></Route>
-            <Route path={routes.sellerListItems} element={<Seller />}></Route>
-            <Route path={routes.about} element={<About_us />}></Route>
-          </Routes>
-        </Router>
+        <Navigation />
+        <Routes>
+          <Route path="/" element={<Home />}></Route>
+          <Route path="/accounts/*" element={<Accounts />}></Route>
+          <Route
+            path={`${import.meta.env.VITE_BACKEND_URL}/auth/google/success`}
+            element={<GoogleLoginSuccess />}
+          ></Route>
+          <Route path="/about_us" element={<About_us />}></Route>
+          <Route path="/accounts/*" element={isLoggedIn() ? <Home /> : <Accounts />}></Route>
```

Comment:
line 25:  `<Route path="/accounts/*" element={<Accounts />}></Route>` is not needed bcz it overwritten by line 31:      `<Route path="/accounts/*" element={isLoggedIn() ? <Home /> : <Accounts />}></Route>` ,

Reply:
Okay spotted!

Reply:
Good! 
But On line 30:  I was just telling you that If you have chosen that way, make sure inside Account Component, there is a path Route that shows sth on` /` (`accounts/`), because we can not ever get 404 page where are /accounts/* is defined, so show something there.

Actually, this Logic on line 30 is not needed. I'd appreciate it if you could think about it again.
first: we don't want to protect the homepage( `<Home />`) which you have well done on line 24 : ` <Route path="/" element={<Home />}></Route>`, so that functions that check that the is logged in or not, is not needed because it is useless, instead to will be used to the protected pages. and it could,  itself direct a user to the login page.